app/channel_test.go

```diff
@@ -1343,15 +1343,25 @@ func TestMarkChannelAsUnreadFromPost(t *testing.T) {
 		require.Nil(t, err)
 		th.CreatePost(c2)
 
+		th.App.CreatePost(&model.Post{
+			UserId:    u2.Id,
+			ChannelId: c2.Id,
+			RootId:    p4.Id,
+			CreateAt:  model.GetMillis() + 2000,
+			Message:   "@" + u1.Username,
+		}, c2, false, true)
+
 		response, err := th.App.MarkChannelAsUnreadFromPost(p4.Id, u1.Id)
 		assert.Nil(t, err)
 		assert.Equal(t, int64(1), response.MsgCount)
```

Comment:
Shouldn't be 2?

Reply:
@jespino surprisingly - no :)
MsgCount = TotalMsgCount - UnreadCount = 3 - 2 = 1